src/ActionRepeater.UI/MainWindow.xaml

```diff
@@ -26,15 +26,35 @@
                             <SolidColorBrush x:Key="TopNavigationViewItemForegroundPointerOver" Color="White" Opacity="0.8"/>
                         </ResourceDictionary>
                     </ResourceDictionary.ThemeDictionaries>
+                    <Style x:Name="MenuItemDescriptorTextBlock" TargetType="TextBlock" BasedOn="{StaticResource BaseTextBlockStyle}">
+                        <Setter Property="VerticalAlignment" Value="Center"/>
+                        <Setter Property="Margin" Value="5 0 0 0"/>
+                    </Style>
                 </ResourceDictionary>
+               
             </NavigationView.Resources>
             
             <NavigationView.MenuItems>
                 <!-- space for the file dropdown button -->
                 <NavigationViewItem x:Name="_navViewOffsetItem" Width="35" IsEnabled="False"/>
                 <!-- binding the tag suddenly stopped working for some fucking reason, so now its hardcoded -->
-                <NavigationViewItem Tag="h" Content="Home"/>
-                <NavigationViewItem Tag="o" Content="Options"/>
+                <NavigationViewItem Tag="h">
+                    <NavigationViewItem.Content>
+                        <StackPanel Orientation="Horizontal">
+                       
+                        <SymbolIcon Symbol="Home"/>
+                            <TextBlock Text="Home" Style="{StaticResource MenuItemDescriptorTextBlock}"/>
+                        </StackPanel>
+                    </NavigationViewItem.Content>
+                </NavigationViewItem>
+                <NavigationViewItem Tag="o">
+                    <NavigationViewItem.Content>
+                        <StackPanel Orientation="Horizontal">
+                            <SymbolIcon Symbol="Setting"/>
+                            <TextBlock Text="Settings" Style="{StaticResource MenuItemDescriptorTextBlock}"/>
+                        </StackPanel>
+                    </NavigationViewItem.Content>
+                </NavigationViewItem>
```

Comment:
```suggestion
                <NavigationViewItem Tag="h">
                    <StackPanel Orientation="Horizontal" Spacing="5">
                        <FontIcon Glyph="&#xE80F;" FontSize="16"/>
                        <TextBlock Text="Home"/>
                    </StackPanel>
                </NavigationViewItem>

                <NavigationViewItem Tag="o">
                    <StackPanel Orientation="Horizontal" Spacing="5">
                        <FontIcon Glyph="&#xE713;" FontSize="16"/>
                        <TextBlock Text="Settings"/>
                    </StackPanel>
                </NavigationViewItem>
```

it seems that theres no way to set the icon size for `SymbolIcon`, so i used a `FontIcon` here. i also kept the default style for the text, and made the icons a bit smaller. looks like this now: 
![image](https://user-images.githubusercontent.com/67976161/234620171-8cd9e367-11d4-40cd-8b0a-4826282259ec.png)